<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	border-image:url(:/background.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(140, 0, 0);
background-color: rgb(247, 255, 178);
font: 87 18pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string>欢迎使用智能充电桩调度计费系统——管理员页面</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="IPLab">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PortLab">
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabWidget:pane {border-top:0px solid #e8f3f9;background:  rgba(0,0,0,170);;}
QTabWidget{
	font: 75 14pt &quot;Arial&quot;;
}</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="QWidget" name="setLab">
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;Arial&quot;;</string>
       </property>
       <attribute name="title">
        <string>1-充电桩情况</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">
font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>       快充电桩数：     </string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>充电桩排队队列长度：     </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>       慢充电桩数：     </string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="WaitingNum">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>暂无</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="ChargingNum">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>暂无</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>     等候区车位容量：      </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="FastNum">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>暂无</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="SlowNum">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>暂无</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="infoLab">
       <property name="styleSheet">
        <string notr="true"> font: 75 12pt &quot;Arial&quot;</string>
       </property>
       <attribute name="title">
        <string>2-充电桩启动/关闭</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="4">
         <widget class="QSpinBox" name="OnNo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QSpinBox" name="OffNo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="Fmode">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>快充</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_8">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>关闭充电桩编号       </string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="4">
         <widget class="QPushButton" name="Off">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 192, 196);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="AllOn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(142, 255, 134);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
          <property name="text">
           <string>一键启动</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_16">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="7">
         <widget class="QPushButton" name="AllOff">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 114, 116);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
          <property name="text">
           <string>一键关闭</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="4">
         <widget class="QPushButton" name="On">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(216, 255, 202);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
          </property>
          <property name="text">
           <string>启动</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="5" colspan="3">
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="7">
         <spacer name="horizontalSpacer_14">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QRadioButton" name="Tmode">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>慢充</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
          </property>
          <property name="text">
           <string>启动充电桩编号       </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5" colspan="3">
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="6" colspan="2">
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reportLab">
       <property name="styleSheet">
        <string notr="true"> font: 75 12pt &quot;Arial&quot;</string>
       </property>
       <attribute name="title">
        <string>3-信息及报表查询</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="Finfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>快充</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);
</string>
            </property>
            <property name="text">
             <string>  查询充电桩状态</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="infoNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetState">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(233, 255, 230);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>确认查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetAllState">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(135, 232, 118);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>一键查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="State">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="Tinfo">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>慢充</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
            </property>
            <property name="text">
             <string>查询排队区车辆信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="CarNo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetInfo">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(233, 255, 230);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>确认查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetAllInfo">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(135, 232, 118);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>一键查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="Info">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
            </property>
            <property name="text">
             <string>  查看报表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetReport">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(135, 232, 118);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>一键查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="Report">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(254, 255, 228);</string>
            </property>
            <property name="text">
             <string>  查看等候区</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="GetWaiting">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(135, 232, 118);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
            </property>
            <property name="text">
             <string>查看</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="waiting">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 200);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="errorLab">
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 10pt &quot;Arial&quot;;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
